apis/standard.oam.dev/v1alpha1/podspecworkload_types.go

```diff
@@ -49,7 +49,7 @@ type PodSpecWorkloadStatus struct {
 // PodSpecWorkload is the Schema for the PodSpec API
 // +genclient:method=GetScale,verb=get,subresource=scale,result=k8s.io/api/autoscaling/v1.Scale
 // +genclient:method=UpdateScale,verb=update,subresource=scale,input=k8s.io/api/autoscaling/v1.Scale,result=k8s.io/api/autoscaling/v1.Scale
-// +kubebuilder:resource:categories={oam}
+// +kubebuilder:resource:categories={oam},shortName=psw
```

Comment:
not necessary, this CRD not used with high frequency

Reply:
OK. Another question is whether `rollout` is still necessary as a short name for `approllout`, because `rollout` maybe has the same concern in common

Reply:
yes, I think we could remove `rollout` and only keep `approllout`

Reply:
Done